src/frontend/scss/generic/_accessibility.scss

```diff
@@ -4,3 +4,45 @@
   position: absolute;
   left: -9999rem;
 }
+
+// Screen reader only content
+.sr-only {
+  border: 0;
+  clip: rect(1px, 1px, 1px, 1px);
+  clip-path: inset(50%);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+  word-wrap: normal !important;
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap;
+  border: 0;
+}
+
+// Screen reader toggable function, including offscreen navigation
+.sr-only-focus:active,
```

Comment:
I'm not sure it's appropriate to use this generic class to specifically style the "skip to main content" link.

IMO this class should just basically reset what `.sr-only` does and set the element back in the flow of the page. We can then use specific classes to create styles individually for these kinds of elements as we want to show them.

Reply:
I'll refactor that. `.sr-only`classes can be available to every element, when needed, but `:focus` is only accessible through element with a `tabindex` defined.